staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -223,8 +223,11 @@ type ServiceAccountSubject struct {
 // ResourcePolicyRule matches a resource request if and only if: (a)
 // at least one member of verbs matches the request, (b) at least one
 // member of apiGroups matches the request, (c) at least one member of
-// resources matches the request, and (d) least one member of
-// namespaces matches the request.
+// resources matches the request, and (d) either (d1) clusterScope is
+// true and the request does not specify a namepace (which happens
```

Comment:
```suggestion
// true and the request does not (or may not) specify a namepace
```

The parenthetical makes this super hard to read, I'd omit it and rely on the field documentation, whether or not you want to include my suggested hint here :)